File: Electronics_Shop_17.Model/Requests/AddOrderItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class AddOrderItem
    {
        public int Quantity { get; set; }
        public decimal Price { get; set; }

        public int OrderId { get; set; }
        public int ProductId { get; set; }
    }
}

File: angular17_web/src/app/models/order-item/add.order.item.ts
export class AddOrderItem {
  quantity!: number;
  price!: number;
  orderId!: number;
  productId!: number;
  finalPrice!: number;
  productColorId!: number;

  constructor(data?: any) {
    if (!data) return;

    this.quantity = data.quantity;
    this.price = data.price;
    this.orderId = data.orderId;
    this.productId = data.productId;
    this.finalPrice = data.finalPrice;
    this.productColorId = data.productColorId;
  }
}
